lib/membrane_ice_plugin/ice_endpoint.ex

```diff
@@ -136,34 +138,40 @@ defmodule Membrane.ICE.Endpoint do
                 [
                   magic: nil,
                   in_nominated_pair: false,
-                  passed_check_from_browser: false,
-                  passed_check_from_sfu: false
+                  passed_check_from_browser: false
                 ]
   end
 
   @impl true
   def handle_init(options) do
     %__MODULE__{
-      ice_lite?: ice_lite?,
       integrated_turn_options: integrated_turn_options,
       dtls?: dtls?,
       handshake_opts: hsk_opts
     } = options
 
-    if ice_lite?, do: Process.send_after(self(), :maybe_send_binding_indication, 1000)
+    Process.send_after(self(), :maybe_send_keepalive, @time_between_keepalives)
```

Comment:
Let's use membrane timer instead https://hexdocs.pm/membrane_core/Membrane.Element.Action.html#t:start_timer_t/0

Also, let's call future timer `keepalive_timer` instead of `maybe_send_keepalive`.
The same for `ice_restart_timeout`

Reply:
You can stop timer using `stop_timer` action